### ISSUE TITLE
Stopped working after updating gems

### ISSUE THREAD

**Issue by felipegomesdev** [TrestleAdmin/trestle#361]
I'm using Ruby 2.5.1 and Rails 5.2

Administrative area with:

- Trestle version 0.8.12;
- Trestle Auth version 0.2.5;
- Trestle Search version 0.3.0;

With this I have normal operation:

![image](https://user-images.githubusercontent.com/6062121/117881949-33d3ab00-b280-11eb-87fd-570a0c6075ce.png)

-----

But when upgrading to versions:

- Trestle version 0.9.5;
- Trestle Auth version 0.4.3;
- Trestle Search version 0.4.3;

With that, the menus stop working, even accessing directly through the url.

![image](https://user-images.githubusercontent.com/6062121/117881997-4352f400-b280-11eb-9a50-a6e10d5bf42c.png)

I don't know what I might be sinning in ...

Any idea?

**Comment by spohlenz** [comment]
Is it possible to upload the source to a demo app somewhere that demonstrates this issue?

I'm struggling to think of what exactly might be causing this, although I think the culprit is either going to be eager loading, or the autoloader (Zeitwerk or classic?)

Are you seeing the same result in production mode as well as development?

**Comment by felipegomesdev** [comment]
Hi, thanks for your interest.

This is a legacy project that still ran the gem **trestle version 0.8.12**. Unfortunately I was not allowed to place it in a demonstration environment.

The autoloader used is the **classic**. I only tested in development mode.
It is a confusing error indeed, nor is there any indication in the logs of the cause of the problem.

But luckily, by locking the **trestle in version 0.9.3** and **trestle-auth 0.3.0**, I was able to solve my problem.
